tests/test_linkify.py

```diff
@@ -656,3 +657,16 @@ def test_only_text_is_linkified(self):
 
         with pytest.raises(TypeError):
             linkify(no_type)
+
+
+@pytest.mark.parametrize('text, expected', [
+    ('abc', 'abc'),
+    ('example.com', '<a href="http://example.com">example.com</a>'),
+    (
+        'http://example.com?b=1&c=2',
+        '<a href="http://example.com?b=1&amp;c=2">http://example.com?b=1&amp;c=2</a>'
+    ),
+])
```

Comment:
nit: might want to add the fragment test from peterbe's branch too https://github.com/peterbe/django-peterbecom/pull/376/files#diff-5a5a4f33d7ed380cb730706e87cb497bR94

Reply:
I like that! I'll do that now.